common/src/main/java/com/wynntils/features/user/redirects/TerritoryMessageRedirectFeature.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user.redirects;
+
+import com.wynntils.core.config.Config;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.notifications.NotificationManager;
+import com.wynntils.mc.event.SubtitleSetTextEvent;
+import com.wynntils.mc.utils.ComponentUtils;
+import com.wynntils.wynn.event.ChatMessageReceivedEvent;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraftforge.eventbus.api.EventPriority;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class TerritoryMessageRedirectFeature extends UserFeature {
```

Comment:
Again, why did we not fold this into InfoMessageFilter?

Reply:
Oh! This is a subtitle event. Okay. Then it is fine to leave it as is. For now. Maybe, we should create an universal feature for titles, like InfoMessageFilter feature is for chat. Not scope of this PR though.

Reply:
Add the \@FeatureInfo annotation though.